### ISSUE TITLE
Can we list any other field beyond "Title" in search results?

### ISSUE THREAD

**Issue by funkybomber** [symbiote/silverstripe-gridfieldextensions#96]
Hi there,

This is not an "issue" per se, I was just wondering if there's some way to list a different field in the search results instead of "Title" (or more than one fields).

The reason I ask for this should be quite obvious, there are many cases where the "Title" is not enough for the user to decide which record is the correct one to pick (many people can share the same name for example). In these cases it would be nice if we could have extra fields listed in the results such as "Code", or "Mobile" etc.

Even if multiple fields cannot be listed due to limitations in the module, a slightly inelegant workaround would be to get all the relevant info by just creating (and populating) a new column in the dataobject with all required info concatenated. For example:
"ExtendedAlternativeTitle" : 
CD0001 - John Smith - 935212541 - johnsmith@gmail.com - 10, Lancelot Str. etc...

I just need to be able to select a different field. Any idea how can I make this work?

Cheers!
![alt_column](https://cloud.githubusercontent.com/assets/13617277/9032542/f316266a-39fe-11e5-9ec9-1d37b851d0f1.png)

**Comment by nyeholt** [comment]
You _could_ provide a custom version of `GridFieldAddExistingSearchHandler.ss` in your project/templates directory.

**Comment by funkybomber** [comment]
Thanks, that was really helpful! Unfortunately I'm not very familiar with Silverstripe. I did create a new .ss file as you said and I changed the "\gridfieldextensions\code\GridFieldAddExistingSearchHandler.php" so that it renders with this new file.
Now however I'm concerned that my change may be reverted in a future update of the module by Composer. 
Is there an easy way to extend the class "GridFieldAddExistingSearchHandler" so that I don't mess with core functionality? I've spent all day trying to make it work but it should be a simple thing.
Thanks again for the useful info!
